tools/api-markdown-documenter/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluid-tools/api-markdown-documenter",
-	"version": "0.10.1",
+	"version": "0.11.1",
```

Comment:
@RishhiB this should really be `0.11.0`. Not a huge problem or anything, but if possible we should fix this before publishing a new version of the library.